packages/client/lib/client/parser.ts

```diff
@@ -0,0 +1,122 @@
+import { RedisArgument, RespVersions } from "../RESP/types";
+import { RedisVariadicArgument } from "../commands/generic-transformers";
+
+export interface CommandParser {
+  redisArgs: ReadonlyArray<RedisArgument>;
+  keys: ReadonlyArray<RedisArgument>;
+  firstKey: RedisArgument | undefined;
+  respVersion: RespVersions;
+  preserve: unknown;
+  cachable: boolean;
```

Comment:
get rid of getter on parser, and just put as part of command struct.

Reply:
addressed on all core commands, never did parser.setCacheable on module commands, will need to determine which need to be cached.